src/core/device.c

```diff
@@ -992,12 +972,20 @@ void device_found_node(Manager *m, const char *node, DeviceFound found, DeviceFo
         if ((found & mask) != 0) {
                 _cleanup_(sd_device_unrefp) sd_device *dev = NULL;
 
+                /* Return early if we're not dealing with a device. */
+                if (!path_startswith(node, "/dev"))
+                        return;
+
```

Comment:
This is gracefully handled by `sd_device_new_from_devname()`, no? Why do you explicitly exclude this?

Reply:
sd_device_new_from_devname() will return EINVAL which results in a lot of log noise. Skipping early here means we don't get log noise from these nodes.

Reply:
Changed to not log when EINVAL is returned by sd-device.